projects/online/online/utils/gdb.py

```diff
@@ -31,7 +31,7 @@ def submit(self, event: Event):
         logging.info(f"Submitting trigger to file {event.filename}")
         event_dir = self.write_dir / f"event_{int(event.gpstime)}"
         filename = event_dir / event.filename
-        response = self.createEvent(
+        response = self.create_event(
```

Comment:
Nice - GraceDB has exposed these pythonic names to the API as well?

Reply:
Yeah, both are valid